src/main.js

```diff
@@ -1,18 +1,55 @@
-const prevButton = document.querySelector('.carousel__buttonPrev--js');
-const nextButton = document.querySelector('.carousel__buttonNext--js');
-const opinionBox = document.querySelector('.carousel__opinions--js');
-const carouselBox = document.querySelector('.carousel__box--js');
+const prevButton = document.querySelector('.js-carousel-prev');
+const nextButton = document.querySelector('.js-carousel-next');
+const opinionBox = document.querySelector('.js-carousel-opinions');
+const carouselBox = document.querySelector('.js-carousel-box');
+const screenBox = document.querySelector('.js-screen-box');
+const hamburger = document.querySelector('.js-hamburger');
+const nav = document.querySelector('.js-nav');
 
+let counter = 0;
+let maxScroll = 0;
+const screenSize = parseInt(screenBox.offsetWidth);
+const position = parseInt(opinionBox.offsetWidth)
+
+function checkScreen(screenSize) {
+    if(screenSize < 768) {
+        maxScroll = 5;
+    } else if (screenSize < 1256) {
+        maxScroll = 5;
+    } else {
+        maxScroll = 3;
+    }
+}
+
+checkScreen(screenSize);
 
 nextButton.addEventListener('click', () => {
-    const position = opinionBox.offsetWidth;
+    checkScreen(screenSize);
```

Comment:
Wtedy on click nie będziesz musiał tego sprawdzać. Zwykle ludzie wchodzą na stronę i nie zmieniają rozmiaru ekranu, to jest głównie case developmentowy ze sobie programista zmienia ekran i to powinno sprawdzać.
Wiec powyższy moj komentarz z "onresize" powinine być wystarczający.